src/OrchardCore.Modules/OrchardCore.Markdown/Drivers/MarkdownBodyPartDisplay.cs

```diff
@@ -73,7 +76,7 @@ private async ValueTask BuildViewModel(MarkdownBodyPartViewModel model, Markdown
             model.ContentItem = MarkdownBodyPart.ContentItem;
             templateContext.SetValue("Model", model);
 
-            var markdown = await _liquidTemplatemanager.RenderAsync(MarkdownBodyPart.Markdown, System.Text.Encodings.Web.HtmlEncoder.Default, templateContext);
+            var markdown = await _liquidTemplatemanager.RenderAsync(MarkdownBodyPart.Markdown, _htmlEncoder, templateContext);
 
```

Comment:
Just to say that here we use a variable `var markdown =` and in the field driver we update the model `model.Markdown =`. Not sure what's the best but maybe better to do the same in both drivers.

A rule would be to set as many things we can to the template context, and then, after having evaluated the liquid expressions, update the model as much as we can.

Sorry to bother you with a so small thing ;) Anyway that's ok for me.